neo4j-jdbc-bolt/src/test/java/org/neo4j/jdbc/bolt/utils/Neo4jContainerUtils.java

```diff
@@ -40,7 +40,7 @@ public static String getVersion(Neo4jContainer<?> neo4j) {
 
     private static String neo4jImageCoordinates() {
         String neo4jVersion = System.getenv("NEO4J_VERSION");
-        if (neo4jVersion == null) neo4jVersion = "4.3";
+        if (neo4jVersion == null) neo4jVersion = "5.6.0";
```

Comment:
maybe default to 4.4 since it's a LTS version

Reply:
ok